core/services/standardcapabilities/standard_capabilities.go

```diff
@@ -74,27 +79,36 @@ func (s *standardCapabilities) Start(ctx context.Context) error {
 			return fmt.Errorf("error starting standard capabilities service: %v", err)
 		}
 
-		if err = s.capabilitiesLoop.WaitCtx(ctx); err != nil {
-			return fmt.Errorf("error waiting for standard capabilities service to start: %v", err)
-		}
+		s.wg.Add(1)
+		go func() {
+			defer s.wg.Done()
+			cctx, cancel := s.stopChan.NewCtx()
```

Comment:
@agparadiso Can we use the CtxWithTimeout variant of this method here and enforce some kind of timeout? Eg. a few minutes should be more than enough.